### ISSUE TITLE
Upgrade neighbourhoods in Pismo Beach, CA

### ISSUE THREAD

**Issue by oeon** [whosonfirst-data/whosonfirst-data#444]
### tl;dr

Status:

- [ ]  Propose sources
- [ ]  Establish concordances between WOF and source
- [ ]  Upgrade / add WOF shapes
- [ ]  Add new or missing features to WOF
- [ ]  Deprecate / supersede WOF features
- [ ]  Draft review
- [ ]  Draft revisions based on review comments
- [ ]  Final review
- [ ]  Final revisions based on review comments
- [ ]  Import into Who's On First

### Details

Sourcing:

* Permissive license?
* Add new src to Who's On First

Upgrade / add:

* `macrohood`
* `neighbourhood`
* `microhood`

Add new or missing features to WOF

* `macrohood`
* `neighbourhood`
* `microhood`

Deprecate / supersede:

* `macrohood`
* `neighbourhood`
* `microhood`

**Comment by oeon** [comment]
I have a source for this data (http://pismobeach.org/106/Development-Standards) and have contacted their Planning department. They do not have the GIS data and would appreciate the effort.

Started work digitizing here: https://gist.github.com/oeon/efe6fc3eef95481d5be4af1e72180371 with process outlined and will post updates.

**Comment by stepps00** [comment]
Hey @oeon! Just checking in to see how things are going with your neighbourhood work. Anything we can help with (Ventura, SLO, or Pismo)?

**Comment by oeon** [comment]
Yo @stepps00 ! Things are going slowly...but my plan is to get some more people involved, as we just started a Maptime chapter locally https://github.com/maptime/maptime-admin/issues/199 and I think this will be a good project.

**Comment by stepps00** [comment]
Fantastic! I'll keep the neighbourhood issues open - feel free to let me know if I can provide you with anymore neighbourhood starter kits.
